docs/introduction/vocabulary.mdx

```diff
@@ -16,4 +16,8 @@ sidebar_position: 3
   flows through the pipeline. It can either change the record or filter it out
   based on some criteria.
 * **Record** - a record represents a single piece of data that flows through a
-  pipeline (e.g. one database row).
\ No newline at end of file
+  pipeline (e.g. one database row).
+* **Collection** - a generic term used in Conduit to describe an entity in a
+  3rd party system from which records are read from or to which records they are
+  written to. Examples are: tables (in a database), indexes (in a search engine),
```

Comment:
I'd also mention topics in case of Kafka, since it's a commonly used connector.

Reply:
Makes sense!